File: Desktop/WpfViewer/C#/WpfViewer/MyCommand.cs
using System;
using System.Windows.Input;
using System.Windows;
using GrapeCity.ActiveReports.Samples.WPFViewer.Properties;

namespace GrapeCity.ActiveReports.Samples.WPFViewer
{
	class MyCommand : ICommand
	{
		public bool CanExecute(object parameter)
		{
			return true;
		}

		public void Execute(object parameter)
		{
			MessageBox.Show(Resources.AboutUsText, Resources.AboutUsCaption, MessageBoxButton.OK);
		}

		public event EventHandler CanExecuteChanged { add { } remove { } }
	}
}

File: Desktop/WpfViewer/VB.NET/WpfViewer/MyCommand.vb
Public Class MyCommand
	Implements ICommand

	Public Function CanExecute(parameter As Object) As Boolean Implements System.Windows.Input.ICommand.CanExecute
		Return True
	End Function

	Public Sub Execute(parameter As Object) Implements System.Windows.Input.ICommand.Execute
		MessageBox.Show(My.Resources.AboutUsText, My.Resources.AboutUsCaption, MessageBoxButton.OK)

	End Sub

	Public Event CanExecuteChanged(ByVal sender As Object, ByVal e As System.EventArgs) Implements System.Windows.Input.ICommand.CanExecuteChanged

End Class
